dozer/cogs/_utils.py

```diff
@@ -1,14 +1,21 @@
 """Utilities for Dozer."""
 import asyncio
 import inspect
+import logging
 import typing
+from asyncore import loop
```

Comment:
Why use `asyncore` here? The module has been deprecated in favor of `asyncio`.

Reply:
Bad autoimport